packages/bitcore-wallet-service/src/lib/server.ts

```diff
@@ -2459,12 +2184,13 @@ export class WalletService {
                     customData: opts.customData,
                     inputs: opts.inputs,
                     version: opts.txpVersion,
-                    fee:
-                      opts.inputs && !_.isNumber(opts.feePerKb)
-                        ? opts.fee
-                        : !ChainService.isUTXOCoin(wallet.coin)
-                        ? opts.fee
-                        : null,
+                    fee: fee
```

Comment:
I think this might be getting too much ternary

Could we break this into a few statements? 

I'm getting this out of it, 

```
if(opts.inputs && !_.isNumber(opts.feePerKb)) {
  fee = opts.fee
} else if(!ChainService.isUTXOCoin(wallet.coin)) {
  fee = opts.fee
} else {
  fee = null
}
```

so if you pull out the booleans

```
const useInputFee = opts.inputs && !_.isNumber(opts.feePerKb);
const isNotUtxoCoin = !ChainService.isUTXOCoin(wallet.coin)
if(useInputFee || isNotUtxoCoin) {
  fee = opts.fee
} else {
  fee = null
}
```

or

```
if(!fee) {
  const useInputFee = opts.inputs && !_.isNumber(opts.feePerKb);
  const isNotUtxoCoin = !ChainService.isUTXOCoin(wallet.coin);
  const shouldUseOptsFee = useInputFee || isNotUtxoCoin;
  fee =  shouldUseOptsFee ? opts.fee : null;
}
```

Reply:
will do